gdal_subsetter/transform.py

```diff
@@ -693,15 +673,10 @@ def reformat(self, transformed_geotiff_file: str, dstdir: str) -> str:
             # * No output specified (GeoTIFF default),
             # * An alternative format that would be handled in downstream
             #   services in the chain.
-            gdal_subsetter_version = f"gdal_subsetter_version={self.service_version}"
             # TODO: Add to consolidated gdal_translate command.
```

Comment:
It's tempting to remove this to-do but it feels a bit naff to just open the file to set a single tag.